Sources/Misc/Obsoletions.swift

```diff
@@ -526,6 +526,115 @@ public extension Purchases {
         fatalError()
     }
 
+    /**
+     * Configures an instance of the Purchases SDK with a specified API key.
+     *
+     * The instance will be set as a singleton.
+     * You should access the singleton instance using ``Purchases/shared``
+     *
+     * - Note: Use this initializer if your app does not have an account system.
+     * ``Purchases`` will generate a unique identifier for the current device and persist it to `NSUserDefaults`.
+     * This also affects the behavior of ``Purchases/restorePurchases(completion:)``.
+     *
+     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
+     *
+     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
+     */
+    @objc(configureWithAPIKey:)
```

Comment:
I don't understand why we need to break this API - this is for folks who don't use observer mode. Why should they have to change anything? Isn't it fine for them to just switch seamlessly from SK1 to SK2?